packages/devtools_app/lib/src/inspector/flutter/inspector_screen.dart

```diff
@@ -104,27 +119,29 @@ class _InspectorScreenBodyState extends State<InspectorScreenBody>
                 minIncludeTextWidth: 800,
               ),
             ),
+            summaryTreeController?.isExperimentalStoryOfLayoutEnabled == true
+              ? OutlineButton(
+              onPressed: summaryTreeController?.toggleDebugLayoutSummaryEnabled,
+              child: Label(
+                FlutterIcons.lightbulb,
+                'Layout',
+                minIncludeTextWidth: 900,
+              ))
+              : const SizedBox(),
             const Spacer(),
             Row(children: getServiceExtensionWidgets()),
           ],
         ),
         Expanded(
           child: Row(
             children: [
+              summaryTree,
               Expanded(
-                  child: InspectorTree(
-                controller: summaryTreeController,
-              )),
-              Expanded(
-                child: Stack(
-                  children: [
-                    InspectorTree(
-                      controller: detailsTreeController,
-                    ),
-                    if (_expandCollapseSupported) _expandCollapseButtons()
-                  ],
-                ),
-              )
+                child: detailsTreeController?.isExperimentalStoryOfLayoutEnabled == true ?
+                  InspectorDetailsTabController(
```

Comment:
I think we could keep the tab controller in both cases but only have 1 tab.
It is a bit confusing currently that we don't label the details tree as the details tree.

Reply:
Nice! I will go ahead and do that then :)